django/contrib/admin/models.py

```diff
@@ -44,6 +51,59 @@ def log_action(
             change_message=change_message,
         )
 
+    def log_actions(
+        self, user_id, queryset, action_flag, change_message="", *, single_object=False
+    ):
+        """
+        Method to bulk_create multiple LogEntry objects from a given queryset.
+        """
+        # If log_action is overridden in a subclass, call it to maintain
+        # backward compatibility
+        if type(self).log_action != LogEntryManager.log_action:
+            warnings.warn(
+                "The usage of log_action() is deprecated. "
+                "Implement log_actions() instead.",
+                RemovedInDjango60Warning,
+                stacklevel=2,
+            )
+            return [
+                self.log_action(
+                    user_id=user_id,
+                    content_type_id=ContentType.objects.get_for_model(
+                        obj, for_concrete_model=False
+                    ).id,
+                    object_id=obj.pk,
+                    object_repr=str(obj)[:200],
```

Comment:
`log_action()` truncates `object_repr` on its own, so there is no need to do this here:
```suggestion
                    object_repr=str(obj),
```

Reply:
done